### PR TITLE
EWSv2 fix html body default behaviour

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-36289)

## Description
Changed logic to return html_body when body is not provided

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
TDD: failing ut in commit [b47259d](https://github.com/demisto/content/pull/34226/commits/b47259d8394e3e118852591a9c6a71afa10e42f5) and this [run](https://github.com/demisto/content/actions/runs/8967915125/job/24626331724?pr=34226)

**Review by kobymeir — APPROVED** [review]

**Review by amshamah419 — APPROVED** [review]
Looks good, great job!

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/README.md [hunk 1]
```diff
@@ -1719,13 +1719,14 @@ Sends an email using EWS.
...
-| body | The contents (body) of the email to send. | Optional | 
-| htmlBody | HTML formatted content (body) of the email to be sent. This argument overrides the "body" argument. | Optional | 
+| body | The contents (body) of the email to send. This argument overrides the "htmlBody" argument if the "bodyType" argument is Text. | Optional | 
+| htmlBody | HTML formatted content (body) of the email to be sent. This argument overrides the "body" argument if the "bodyType" argument is HTML. | Optional | 
 | attachIDs | A CSV list of War Room entry IDs that contain files, and are used to attach files to the outgoing email. For example: attachIDs=15@8,19@8. | Optional | 
 | attachNames | A CSV list of names of attachments to send. Should be the same number of elements as attachIDs. | Optional | 
 | attachCIDs | A CSV list of CIDs to embed attachments within the email itself. | Optional | 
 | raw_message | Raw email message from MimeContent type. | Optional | 
 | from | The email address from which to reply. | Optional | 
+| bodyType | Whether message response body type is Text, or HTML. Default is Text. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/README.md:1729]:
```suggestion
| bodyType | Whether message response body type is Text or HTML. Default is Text. | Required | 
```

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_6.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Fixed an issue where providing `htmlBody` and not providing `body` arguments to the ***send-mail*** command resulted in an empty email body for the default `bodyType`.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_6.md:6]:
```suggestion
Fixed an issue where providing `htmlBody` and not providing `body` arguments to the ***send-mail*** command resulted in an empty email body for the default `bodyType`.  
```

**Comment by ShirleyDenkberg** [comment]
@kobymeir @amshamah419

**Pull request merged**